src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": [
+    {
+      "name": "application.data-generation.time-freq",
+      "type": "java.lang.String",
```

Comment:
It has to be a integer value.

Reply:
Also, we need add _spring-boot-configuration-processor_ to dependencies within pom.xml file.